tests/test_database.py

```diff
@@ -319,3 +319,20 @@ def test_database_utf8(sys_db, db_version, special_db_names):
         assert sys_db.create_database(name)
         assert sys_db.has_database(name)
         assert sys_db.delete_database(name)
+
+
+def test_license(sys_db, db, enterprise):
```

Comment:
todo: need to add a test case for `set_license`

Reply:
added for community edition: [a416b18](https://github.com/ArangoDB-Community/python-arango/pull/285/commits/a416b18b8ff766bde5288e8fcc30bed2b39e1bd3)